src/main/java/graphql/GraphqlErrorBuilder.java

```diff
@@ -59,22 +59,32 @@ public GraphqlErrorBuilder message(String message, Object... formatArgs) {
     }
 
     public GraphqlErrorBuilder locations(List<SourceLocation> locations) {
-        this.locations.addAll(assertNotNull(locations));
+        if (locations != null) {
```

Comment:
can we add the Nullable annotation to the argument to make this more clear?

Reply:
@bbakerman I think that is a missunderstanding: the result is never null, but the argument. So the directive should be on the argument.

Reply:
done